docs/content/concepts/logging/python-logging.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Python Logging | Dagster
+description: Dagster is compatible and configurable with Python's logging module.
+---
+
+# Python Logging
+
+Dagster is compatible and configurable with [Python's logging module](https://docs.python.org/3/library/logging.html). You can specify and configure handlers and formatters to create custom log messages.
+
+## Configuring Python Logging
+
+In your config YAML, you can configure handlers and formatters that will apply to the Dagster instance, so all pipeline runs will share the same logging configuration.
+
+```yaml file=/concepts/logging/python_logging_config.yaml
+python_logs:
+  dagster_handler_config:
+    handlers:
+      myHandler:
+        class: logging.StreamHandler
+        level: INFO
+        stream: ext://sys.stdout
+        formatter: myFormatter
+    formatters:
+      myFormatter:
+        format: 'My formatted message: %(message)s'
+```
+
+Handler and formatter configuration follows the [dictionary config schema format](https://docs.python.org/3/library/logging.config.html#logging-config-dictschema) in the Python logging module. Only the
+`handlers` and `formatters` dictionary keys will be accepted, as Dagster creates loggers internally.
+
+From there, standard `context.log` calls will output with your configured handlers and formatters.
```

Comment:
I think to make the value a little clearer / more searchable, I think it might be cool to have a sub-heading called "Example: Outputting all Dagster logs to a file.", and then explain "first, we can use the built-in python logging.FileHandler, and specify that under the handlers section (etc.)". If it's easy, showing a JSON formatter as an option would also be nice.

These sorts of examples are super useful especially for people not particularly familiar with the python logging ecosystem, so having an example they can follow along with / mostly copy and paste makes a big difference. I also think shifting to a file handler makes it a bit more obvious why you might want to use this sort of feature.

Reply:
Thanks, added this example!